calm-extension/src/extension.ts

```diff
@@ -0,0 +1,77 @@
+import * as vscode from 'vscode';
+import { visualize } from './visualize.js';
+
+export function activate(context: vscode.ExtensionContext) {
+    let panel: vscode.WebviewPanel | undefined = undefined;
+    let svg: string | undefined;
+
+    context.subscriptions.push(
+        vscode.commands.registerCommand('calm.visualize.instantiation', async () => {
```

Comment:
So how does this behave? Do we load into a `.json` and then run the `Calm Visualize Instantiation File` command from the palette.  It loads the extra panel - does this panel update as we navigate files or will it show whatever was initially rendered? 

Obviously all of this can be improved in time, just curious how you envisioned it working.

Reply:
VS Code provides an API that enables extensions to read values from an active text or JSON tab and visualize the diagram in a new tab. The visualization panel will automatically update whenever the user saves changes to the file.